cmake_modules/TokuSetupCompiler.cmake

```diff
@@ -84,14 +84,15 @@ macro(set_ldflags_if_supported)
 endmacro(set_ldflags_if_supported)
 
 ## disable some warnings
+## missing-format-attribute causes warnings in some MySQL include files
+## if the library is built as a part of TokuDB MySQL storage engine
```

Comment:
Including this flag for mysql storage engine causes compilation errors (there is no such flag for mysql build). Besides, there is gcc bug report https://gcc.gnu.org/bugzilla/show_bug.cgi?id=28492 which takes place for our build. But if you really think that reducing the code readability by introducing one more "if" and copy-paste in .cmake file for remote possibility for the library to be built outside of the storage engine, I will fix it.